packages/donor/src/components/Wizard/WizardScreen.tsx

```diff
@@ -1,40 +1,103 @@
-import Stepper from "../basic/Stepper";
-import Button, { ButtonVariant } from "../basic/Button";
+import Button from "../basic/Button/ButtonV2";
+import Text from "../basic/Text";
+import styles from "./WizardScreen.module.scss";
+import { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import MobileStepper from "@material-ui/core/MobileStepper";
+import { BrowserRouter, Link } from "react-router-dom";
+import { PropTypes } from "@material-ui/core";
+import { ButtonVariant } from "../basic/Button";
 
-export interface WizardScreenProps {
-  imageUrl: string;
-  title: string;
-  content: string;
+const useStyles = makeStyles({
+  root: {
+    flexDirection: "column",
+    fontFamily: "Rubik",
```

Comment:
The font should be set in the app, and not here. 
If we ever want to change the app font, we should only change it in one place

Reply:
removed it